examples/models/question_answering/Question_Answering.py

```diff
@@ -1,695 +0,0 @@
-from __future__ import division
```

Comment:
what's the difference to QuestionAnswering.py?

Reply:
1. I worked around the train process to avoid model train as needed.
2. As last time we communicated, dev.py has to be changed to comply with QA's special case, this model is stable with updated dev.py and admin.py in Singa-Auto pr#80
3. I optimised its return format, to comply with Singa-Auto inference worker

Reply:
you have QuestionAnswering.py and Question_Answering.py.
What's the difference?

Reply:
Question_Answering.py is outdated. I suggest we remove it and replace it with singa-easy pr #35